anax-in-k8s/Dockerfile.ubi.auto-upgrade-cron.amd64

```diff
@@ -14,7 +14,7 @@ RUN  microdnf update  -y libxml2 --nodocs --setopt=install_weak_deps=0 --disable
   && microdnf install -y --nodocs --setopt=install_weak_deps=0 --disableplugin=subscription-manager ${REQUIRED_RPMS} \
   && microdnf clean all --disableplugin=subscription-manager \
   && rm -rf /mnt/rootfs/var/cache/* /mnt/rootfs/var/log/dnf* /mnt/rootfs/var/log/yum.* \
-  && curl -4LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl \
+  && curl -4LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://dl.k8s.io/release/stable.txt`/bin/linux/amd64/kubectl \
```

Comment:
This wasn't successful. I tried executing this
```
curl -s https://dl.k8s.io/release/stable.txt
```
and got
```
<html>
<head><title>302 Found</title></head>
<body>
<center><h1>302 Found</h1></center>
<hr><center>nginx</center>
</body>
</html>
```


If we are going to get stable.txt from dl.k8s.io, then the command should be
```
curl -4LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" 
```